src/components/SubscriberListSelector/NoList/NoList.js

```diff
@@ -0,0 +1,45 @@
+import { useIntl } from 'react-intl';
+import { FormattedMessageMarkdown } from '../../../i18n/FormattedMessageMarkdown';
+import React from 'react';
+
+export const NoList = ({ onCancel }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  return (
+    <div className="dp-container" data-testid="no-list">
+      <div className="dp-rowflex">
+        <div className="col-sm-12">
+          <div className="dp-icon-modal">
+            <img src={_('common.ui_library_image', { imageUrl: 'lists.svg' })} alt="icon" />
```

Comment:
instead of alt = "icon", can you replace it with something more descriptive?

Reply:
sure, we can replace it by `alt="lists logo" `